filetests/isa/x86/legalize-i64.clif

```diff
@@ -0,0 +1,16 @@
+; Test the legalization of i64 instructions on x86_32.
+test legalizer
+target i686 haswell
+
+; regex: V=v\d+
```

Comment:
Maybe this regex should be backed into filetest itself, as it is used often.

Reply:
@bnjbvr

Reply:
I wonder if this wouldn't be too magical, I'm fine with the current situation.